shared/js/background/safari-events.es6.js

```diff
@@ -336,6 +295,14 @@ let onClose = (e) => {
 // update blocking lists on interval
 setInterval(abpLists.updateLists, 30 * 60 * 1000)
```

Comment:
We should prevent the extension from trying to download stuff. Can we also delete the initial `updateLists()` call in `abpLists`?